docs/user-manual/software/cltool.md

```diff
@@ -153,31 +154,71 @@ OPTIONS (RTK Rover / Base)
 Windows Visual Studio supports CMake projects. Follow the instructions provided by Microsoft:
 https://learn.microsoft.com/en-us/cpp/build/cmake-projects-in-visual-studio?view=msvc-170
 
-## Update the Firmware
-In order to update the firmware of your unit on the CLTool, follow these steps:
+## Updating Firmware with CLTool
+### Updating using Firmware Package
 
-1. Navigate to the directory with the CLTool executable
-2. Set the unit's COM port as an option, e.g. `-c COM15`
-3. Specify the FILEPATH to the .hex file, e.g. `-uf foo/bar/IS_uINS-3.hex`
-4. Optionally specify the bootloader BLFILEPATH to the .bin file, e.g. `-ub foo/bar/bootloader-SAMx70.bin`
-5. Run the executable
+Updating firmware using a firmware package file provides a simple method to update multiple devices in one process.  This include the ability to update an IMX-GPX module pair in one step.  The cltool only needs know the file path of the firmware package file and the serial port of the device to be updated.  The file extension for a firmware package is `.fpkg`.
 
-*Note: The firmware can only be updated at the following baud rates: 300000, 921600, 460800, 230400, 115200
+```bash
+cltool -c DEVICE_PORT -ufpkg FILEPATH
+```
 
-## Logging
+The following is a specific example using a firmware package file:
 
-1. Make sure you have followed the steps shown above to compile your CLTool.
-1. Change your current directory to `/cpp/SDK/cltool/build`
-1. Type `./cltool` into the command line while appending your desired options (All possible options can be accessed from the CLTool’s help menu which is accessed by entering `./cltool -h` into the command line)
-1. The options you will minimally need to log are these:
-	* `-lt=#` (Defines the log type. Either e.g. -lt=dat or -lt=csv)
-	* `-lp /directory1/directory2/directory3` (Specifies the path into which your files will be placed.)
-		* If you don’t include this option, your data will be saved to `/build/IS_logs` if that directory has already been created.
-	* `-lon` (Must be placed after all other options specified)
-1. This is an example of what you could use as your logging options:
-   ```
-   cltool -lon -lts=1 -lp /media/usbdrive/data
-   ```
+```bash
+cltool -c /dev/ttyACM0 -uf-cmd "package=IS-firmware_2.0.3_2024-03-18_213925.fpkg"
+```
+
+### Updating using Single Firmware File
+
+The CLTool can be used to update device firmware with the following options:  
```

Comment:
This should be updated to reflect that this is the "Legacy" mode, and is only applicable on the IMX-5 and earlier products (uINS3, EVB, etc).